ci/create_docker_tag.sh

```diff
@@ -0,0 +1,21 @@
+DOCKER_REPO="semitechnologies/weaviate"
```

Comment:
please add shebang to the sh script, and also set a script to propagate exit codes
```
#!/usr/bin/env bash

set -euo pipefail
```